Cauli/Florets/Inspector/InspectorFloret.swift

```diff
@@ -31,6 +31,8 @@ import UIKit
 /// * filter `Record`s by the request URL.
 public class InspectorFloret: DisplayingFloret {
 
+    public var description: String? = "Tap to view records captured when the InspectorFloret is active."
```

Comment:
Please comment better descriptions for the `InspectorFloret`

Reply:
Let me give a shot at writing a description 😊

> Tap to inspect network request and responses. Data is recorded as long as Cauli is enabled.